rules/integrations/azure/privilege_escalation_azure_kubernetes_rolebinding_modified_or_deleted.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/10/18"
+integration = "azure"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies the modification or deletion of potential malicious rolebinding. You can assign these roles to Kubernetes subjects 
+(users, groups, or service accounts) with role bindings and cluster role bindings.
```

Comment:
Needs elaboration, why is this potentially malicious?

Reply:
Also, in this context, the second part still makes sense? As you don't assign to delete something